guake/tests/test_guake.py

```diff
@@ -1,2 +1,106 @@
-def test_000():
-    pass
+# -*- coding: utf-8 -*-
+# pylint: disable=redefined-outer-name
+
+import json
+import os
+import pathlib
+import tempfile
+
+import pytest
+
+import guake.guake_app
+from guake.guake_app import Guake
+
+
+@pytest.fixture
+def g(mocker):
+    mocker.patch(
+        'guake.guake_app.Guake.get_xdg_config_directory',
+        return_value=pathlib.Path(tempfile.gettempdir())
```

Comment:
Better, you can use [fakefs](https://pypi.org/project/pyfakefs/)

Reply:
I still stuck with `pathlib.Path`, since most of the `Path` object in the code is using `pathlib`, and using `path.py` make some breakage, e.g. `Path().exists()` (pathlib) -> `os.path.exists(Path())` (path.py), `with Path().open() as f` (pathlib) -> `with open(Path()) as f` (path.py)

Reply:
Address it with pyfakefs